### DISCUSSION TITLE
Multi language store + Multi language store

### DISCUSSION THREAD

**Discussion by f1-outsourcing** [OpenMage/magento-lts#3429] [Questions & Answers]
I think the general advice of implementing multiple languages for a store is through the use of multiple store views. However it looks like the store view code is set globally unique. Should website name + store view code not be the unique identifier? Currently I am having to choose weird store codes. (which show in the url)

Is there are a way around this, so I can keep 'en_gb' code for all the UK stores?
 
![store-view](https://github.com/OpenMage/magento-lts/assets/13450219/8720e8c8-8bad-4a10-b854-cee32fac300c)

https://www.cloudways.com/blog/magento-multi-language/
https://magento.stackexchange.com/questions/128286/using-url-subfolders-for-multi-language-store-views-in-magento-2

**Comment by fballiano** [comment]
not the cleanest but I'd add a "website" prefix to the store view code
